---
type: openai
client_config:
  model: gpt-4o
use_cuda: true
clip_model:
  type: open_clip
  model_name: ViT-B-32  # ViT-L/14, ViT-B/32
  tokenizer_name: ViT-B-32
  pretrained: laion2b_s34b_b79k
